wolfssl/test.h

```diff
@@ -853,8 +852,12 @@ static INLINE void tcp_listen(SOCKET_T* sockfd, word16* port, int useAnyAddr,
     if (bind(*sockfd, (const struct sockaddr*)&addr, sizeof(addr)) != 0)
         err_sys("tcp bind failed");
     if (!udp) {
-        if (listen(*sockfd, 5) != 0)
-            err_sys("tcp listen failed");
+        #ifndef WOLFSSL_KEIL_TCP_NET
+            if (listen(*sockfd, 5) != 0)
```

Comment:
Would you mind making this 5 a define. Something like `#define SOCK_LISTEN_MAX_QUEUE 5` and using it here in the function call.

```
#ifndef SOCK_LISTEN_MAX_QUEUE
    #ifdef WOLFSSL_KEIL_TCP_NET
        #define SOCK_LISTEN_MAX_QUEUE 1
    #else
        #define SOCK_LISTEN_MAX_QUEUE 5
#endif
```

Reply:
Fixed